composeApp/src/commonMain/kotlin/com/github/familyvault/ui/screens/main/familyGroupSettings/familyGroupMember/ModifyFamilyMemberScreen.kt

```diff
@@ -76,6 +80,11 @@ class ModifyFamilyMemberScreen(private val familyMember: FamilyMember) : Screen
                 FamilyGroupMemberPermissionGroup.Guest
             )
         )
+
+        LaunchedEffect(Unit) {
+            familyMembers.addAll(familyGroupService.retrieveFamilyGroupMembersList())
```

Comment:
Dać może tutaj jakiś loader, że zanim się pobiorą dane wszystkich członków rodziny to musimy zaczekać. Bo może to prowadzić do błędów jeżeli będziemy operowali dalej a dane się nie załadują.